script/openqa-clone-custom-git-refspec

```diff
@@ -119,7 +119,11 @@ Please try 'curl $json_url' or select another job, e.g. in the same scenario: $h
         old_productdir=$(echo "$json_data" | jq -r '.PRODUCTDIR') || throw_json_error "$json_url" "$json_data"
         local old_casedir
         old_casedir=$(echo "$json_data" | jq -r '.CASEDIR') || throw_json_error "$json_url" "$json_data"
-        local productdir="${productdir:-"${repo_name##*/}${old_productdir#*${old_casedir##*/}}"}"
+        if [[ ${old_productdir:0:1} == "/" ]]; then
+            local productdir="${productdir:-"${repo_name##*/}${old_productdir##$old_casedir}"}"
+        else
+            local productdir="${productdir:-"${repo_name##*/}${old_productdir#*${old_casedir##*/}}"}"
+        fi
```

Comment:
May I suggest you add a case to `t/40-script_openqa-clone-custom-git-refspec.t`?

Reply:
Added